boards/nucleo144-f413/include/periph_conf.h

```diff
@@ -70,6 +70,27 @@ extern "C" {
 /** @} */
 
 /**
+ * @name    DMA streams configuration
+ * @{
+ */
+static const dma_conf_t dma_config[] = {
```

Comment:
I don't see why this should be part of the board configuration. AFAIK the mapping of DMA streams to peripherals is static for each CPU, so there should be no need to configure this on a per-board basis.

Reply:
This is not that easy, because some peripheral can use different streams and two (or more) peripheral can use the same stream.
Then, we need a mutex per used stream and if we do not want to allocate mutexes for stream which are unused, we need a way to know which streams will be used.
The easiest way I could think was to declare this array in the `periph_conf`. But I would be happy to get rid of it!